app/views/tutors/index.html.erb

```diff
@@ -1,15 +1,51 @@
 <h2>Tutors</h2>
 
-<table>
+<table class="table table-striped">
+  <thead class="thead-inverse">
   <tr>
-    <th>Title</th>
-    <th>Text</th>
+    <th>First Name </th>
+    <th>Last Name </th>
+    <th>Gender </th>
+    <th>Date of Birth </th>
+    <th>Address </th>
+    <th>Home Phone </th>
+    <th>Cell Phone </th>
+    <th>Email Preferred</th>
+    <th>Email Other</th>
+    <th>Native Language </th>
+    <th>Orientation Date </th>
+    <th>Country of Birth</th>
+    <th>Training Date </th>
+    <th>Training Type </th>
+    <th>Referral </th>
+    <th>Education </th>
+    <th>Employment status </th>
+    <th>Specific Occupation </th>
```

Comment:
I don't think all of these are necessary for the index view. Usually index views only show a few choice attrributes.. Maybe first&last name, email, and phone.. Everything else can be viewed in the show view. Ideally it will easily fit on one page, horizontally.